app/main.py

```diff
@@ -1,68 +1,71 @@
+from __future__ import annotations
+
+from typing import Union
+
+
 class Distance:
-    def __init__(self, km: float):
+    def __init__(self, km: Union[int, float]) -> None:
         self.km = km
 
-    def __str__(self):
+    def __str__(self) -> str:
         return f"Distance: {self.km} kilometers."
 
-    def __repr__(self):
+    def __repr__(self) -> str:
         return f"Distance(km={self.km})"
 
-    def __add__(self, other):
+    def __add__(self, other: Union[Distance, int, float]) -> Distance:
         if isinstance(other, Distance):
             return Distance(self.km + other.km)
         else:
             return Distance(self.km + other)
 
-    def __iadd__(self, other):
+    def __iadd__(self, other: Union[Distance, int, float]) -> Distance:
         if isinstance(other, Distance):
             self.km += other.km
             return self
         else:
             self.km += other
             return self
```

Comment:
return self is duplicated, take it outside if-else

Reply:
Done)